docs/root/_include/cert_stats.rst

```diff
@@ -0,0 +1,5 @@
+.. csv-table::
+   :header: Name, Type, Description
+   :widths: 1, 1, 2
+
+   ``expiration_unix_time``, Gauge, Number of seconds since UNIX epoch of the expiration date of the certificate.
```

Comment:
Suffix with `_seconds` to this name for clarity?

Reply:
sg, 
renamed to `expiration_unix_time_in_seconds`